app/_landing_pages/konnect-reference-platform.yaml

```
@@ -0,0 +1,114 @@
+metadata:
+  title: "{{site.konnect_short_name}} Reference Platform"
+  content_type: landing_page
+  products:
+    - api-ops
+  description: |
+    Provides a complete guide for platform builders to to integrate {{site.konnect_product_name}} into their engineering 
+    organization and API delivery process. 
+  tags:
+    - konnect
+    - automation
+    - apiops
+  breadcrumb: /konnect-reference-platform/
+
+rows:
+  - header:
+      type: h1
+      text: "{{site.konnect_short_name}} Reference Platform"
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    The {{site.konnect_short_name}} Reference Platform is a technical guide designed to help you integrate 
+                    [{{site.konnect_product_name}}](https://konghq.com/products/kong-konnect)
+                    into your engineering organization's technology stack. The platform provides an opinionated framework that will aid you in 
+                    designing your {{site.konnect_short_name}} organization and the building of a _Federated APIOps_ delivery process with 
+                    governance, auditing, and GitOps based automations. The platform provides tools, examples, best practices, and documentation 
+                    to guide you in on-boarding with {{site.konnect_short_name}}. You can choose how to use the components, 
+                    individually or collectively, to fit your needs and engineering requirements.
+                - type: text
+                  text: |
+                    This document outlines a suggested learning path for using the platform. Start by reading the FAQ section to 
+                    understand its components and their purpose. Each component is then explained in detail, with relevant links to 
+                    guide you on their usage. You can use individual components as needed, but if you decide the full platform is a 
+                    good fit for your organization, follow the how-to guide to set 
+                    up an instance connected to your {{site.konnect_short_name}} account and code repositories.
+
+  - header:
+      type: h2
+      text: "Frequently Asked Questions"
```

```suggestion
      text: "Frequently asked questions"
```
We're using sentence case for headings (except on index pages)